packages/netsuite-adapter/src/filters/element_references.ts

```diff
@@ -254,6 +266,30 @@ const applyValuesAndAnnotationsToElement = (element: Element, newElement: Elemen
   element.annotations = newElement.annotations
 }
 
+export const extractCustomRecordFields = (customRecordType: ObjectType): ElemServiceID[] =>
+  Object.values(customRecordType.fields)
+    .map(field => ({ serviceID: getServiceId(field), elemID: field.elemID }))
```

Comment:
We usually set the references to the `scriptid` value in the element
```suggestion
    .map(field => ({ serviceID: field.annotations[SCRIPT_ID], elemID: field.elemID.createNestedID(SCRIPT_ID) }))
```